### ISSUE TITLE
No close/OK/Done/... button on Preferences and About popups

### ISSUE THREAD

**Issue by keikoro** [toggl/toggldesktop#2751]
### 💻 Environment

Platform: Linux
OS Version: Debian GNU/Linux 9 (stretch)
Toggl Version: 7.4.272

### 🐞 Actual behavior
The Preferences  and About dialogues do not seem to have any kind of obvious button with which to close them and also do not react to CTRL+W. The only way I could close them was via the right-click context menu.

### 💯 Expected behavior
Either the usual set of menu buttons – minimise, maximise, close – (at least the last one out of those three), or an otherwise obvious button to close the dialogue(s).

I don't remember this to have been a problem before (?), but will have to check on another machine which is still running an older version of Toggl.

### 🔨 Steps to reproduce

1. Open Preferences (or About) menu
2. Look for an obvious way to close it
3. Don't find one

### 📦 Additional info

![screenshot from 2019-01-10 01-29-45](https://user-images.githubusercontent.com/559572/50938516-d6917a00-1478-11e9-9869-22b9bb6601cc.png)

**Comment by IndrekV** [comment]
This issue seems to be specific for the window manager. I'm using KDE and I have the header buttons as expected.

**Comment by keikoro** [comment]
@IndrekV Ah. I currently don't have access to this particular machine, but will check back in about this issue next week.

**Comment by MartinBriza** [comment]
Yeah, older versions of GNOME didn't have close buttons on dialog windows for some reason. You can still close it by using alt+f4
